sdk/src/Handlers/AspNet/AWSXRayRecorder.Handlers.AspNet.csproj

```diff
@@ -29,8 +29,8 @@
     <NoWarn>1701;1702;1705;1591;1587;1572;1573</NoWarn>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.3.25.1" />
+  <ItemGroup>
```

Comment:
If possible, can we avoid this file updated, since there is no diff.

Reply:
I thought I caught this, I will handle it.

Reply:
Gentle reminder, this is still pending.

Reply:
Fixed!